docs/index.md

```diff
@@ -0,0 +1,65 @@
+---
+id: index
+title: "Overview"
```

Comment:
To make a consistent title with other projects:

title: Introduction to ZIO DynamoDB
sidebar_title: ZIO DynamoDB